src/frontend/src/services/tmdbService.js

```diff
@@ -30,6 +32,7 @@ export async function getTMDBImage(imdb_id, size) {
     const person = val.person_results[0];
     path = person.profile_path;
     tmdbId = person.id; // Set TMDBID from person results
+    knownFor = person.known_for;
```

Comment:
I'm a little confused why known for is returned from a function that fetches a image url? I think we should have this logic not be in a separate function for fetching known titles.

Also why is tmdbId returned?